manifests/init.pp

```diff
@@ -38,7 +38,7 @@
   $vas_conf_locked_out_pwhash                           = undef,
   $vas_conf_preload_nested_memberships                  = 'UNSET',
   $vas_conf_update_process                              = '/opt/quest/libexec/vas/mapupdate_2307',
-  $vas_conf_upm_computerou_attr                         = 'department',
+  $vas_conf_upm_computerou_attr                         = 'UNSET',
```

Comment:
This would change the default behaviour of the module and breaks backward compatibility by that.